### ISSUE TITLE
out of control memory use on rhombus/private/string.rhm

### ISSUE THREAD

**Issue by samth** [racket/drracket#613]
If I open `rhombus/private/string.rhm` from the `rhombus-prototype` repository, DrRacket quickly completes background expansion but memory use (as reported in the bottom corner) continues growing without bound (eg from 800MB to 8GB in a couple minutes). Nothing appears to be happening while this memory usage growth happens, and it hasn't happened with other Rhombus programs that I've noticed.

**Comment by rfindler** [comment]
I see memory use climb but then drop back down again.

**Comment by samth** [comment]
How high does it climb? For me I have to close the tab or eventually close DrRacket.

**Comment by rfindler** [comment]
Not very high! Maybe 20mb before dropping back down?

**Comment by samth** [comment]
Right, for me it grows by gigabytes. I wonder if I have some different configuration that might be at issue?

**Comment by rfindler** [comment]
The language settings can affect online expansion. It might not actually be related to online expansion, tho.

Can you get a stack trace? Perhaps control-c in the terminal window where drr is started?

**Comment by samth** [comment]
I think I had another file that used Rhombus open, and a change in private/string.rkt (relative to the zo version). I'm working on getting more precise reproduction instructions.
